package.json

```diff
@@ -27,7 +27,14 @@
     "require": [
       "babel-register",
       "babel-polyfill"
-    ]
+    ],
+    "babel": {
+      "presets": [
+        "es2015",
+        "stage-3",
+        "react"
+      ]
+    }
```

Comment:
ok you can get rid of the duplicate babel configuration and just add `babel: "inherit"`

```json
"ava": {
  "require": [
    "babel-register",
    "babel-polyfill"
  ],
  "babel": "inherit"
},
```